backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -235,15 +242,15 @@ private Entity.Parcel AddUpdateParcel(PropertyModel property, int pid, Entity.Ag
                     throw new InvalidOperationException($"Agency '{property.Agency}' does not exist.");
                 p_e.Agency = agency;
                 p_e.Description = property.Description;
-                p_e.Latitude = double.Parse(property.Latitude);
-                p_e.Longitude = double.Parse(property.Longitude);
-                p_e.LandArea = float.Parse(property.LandArea);
+                p_e.Latitude = String.IsNullOrWhiteSpace(property.Latitude) ? 0 : double.Parse(property.Latitude);
+                p_e.Longitude = String.IsNullOrWhiteSpace(property.Longitude) ? 0 : double.Parse(property.Longitude);
+                p_e.LandArea = String.IsNullOrWhiteSpace(property.LandArea) ? 0 : float.Parse(property.LandArea);
```

Comment:
I like that we are checking data before we put it in the database but the above does not protect the server from bad data, e.g. LandArea = "$1,634". Such value would cause `float.Parse` to throw an error. Have you considered using "double.TryParse()` instead?

Reply:
I've updated to use the `TryParse`